test/libsolidity/smtCheckerTests/special/msg_sig.sol

```diff
@@ -26,5 +26,5 @@ contract C
 }
 // ----
 // Warning 6328: (76-105): CHC: Assertion violation happens here.\nCounterexample:\n\n\nTransaction trace:\nC.constructor()\nC.f()
-// Warning 6328: (403-432): CHC: Assertion violation happens here.\nCounterexample:\n\n\nTransaction trace:\nC.constructor()\nC.f()\n    C.fi() -- internal call\n    C.gi() -- internal call
+// Warning 6328: (403-432): CHC: Assertion violation happens here.\nCounterexample:\n\n\nTransaction trace:\nC.constructor()\nC.f()\n    C.gi() -- internal call\n    C.fi() -- internal call
```

Comment:
Do you know why the order has changed here?
Shouldn't the order in report match the order in the call already? (I forgot if this has been  implemented or not)

Reply:
Oh true, good catch. It should be implemented, maybe the order here impacted that somehow, I'll check.